src/tradingkit/data/feed/kraken_feeder.py

```diff
@@ -26,13 +26,15 @@ class KrakenFeeder(Feeder, Publisher):
             "BTC/EUR": "XBT/EUR",
             "BTC/USD": "XBT/USD",
             "BTC/USDT": "XBT/USDT",
+            "ETH/BTC": "ETH/XBT",
```

Comment:
must create separate PR's for different change types

Reply:
done